webapp/components/create_comment.jsx

```diff
@@ -150,6 +151,8 @@ class CreateComment extends React.Component {
         Client.createPost(
             post,
             (data) => {
+                AsyncClient.getPosts(this.props.channelId);
```

Comment:
Actually this is the way it was always working and I removed them on my last related PR but it broke other things so I added them back.

And the actual fix is the `isPost` flag for the `PostStore`